rpm/generic/zfs.spec.in

```diff
@@ -99,6 +99,13 @@
 %define __python_cffi_pkg         python%{__python_pkg_version}-cffi
 %define __python_setuptools_pkg   python%{__python_pkg_version}-setuptools
 %endif
+%define __py_major_version        %(%{__python} -c "import sys; print(str(sys.version_info[0]))")
+%if 0%{?__py_major_version} >= 3
+%define __python_sitelib          %(%{__python} -Ic "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
```

Comment:
Instead of using `-Ic`, use `-Esc` and then we don't have to care about the `%__py_major_version` and that logic can be dropped.

We don't need to reference `%python_sitelib` at all either.

Reply:
Even better .. Thanks!